package.json

```diff
@@ -43,6 +43,7 @@
     "@types/morgan": "^1.9.3",
     "@types/multer": "^1.4.7",
     "@types/node": "^16.6.1",
+    "@types/nodemailer": "^6.4.4",
```

Comment:
Is this correct? We have the `@types/nodemailer": "^6.4.4` dependency repeated on dev dependencies and also on the dependencies.

Reply:
done, thanks !